### PR TITLE
Add Circulation Cover Page template

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
*As requested in https://github.com/metanorma/metanorma-iec/issues/95*

### CODE REVIEW

**Comment by manuelfuenmayor** [comment]
Hi @opoudjis,

I draw your attention to some questions that came up while creating the template. Some are related with a sample you shared in another thread (https://github.com/metanorma/metanorma-iec/files/5750501/coverpage.zip):

1. Which attribute would set the stage of the standard (e.g. FDIS, CDV, RVC, etc.) ?
2. In the sample you shared, the `:stage:` attribute has a number as a value. But according to the documentation, it should be an abbreviation.
3. Has the "Supersedes Documents" field any attribute associated?

**Comment by manuelfuenmayor** [comment]
Hey @opoudjis, thank you for your answers in https://github.com/metanorma/metanorma-iec/issues/95.
I have applied minor updates to the attributes. Could you take a brief look to see if it's all good?
Thanks!

File: cover_page/document.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+= Title of IEC standard
+:comment: ### Circulation Cover Page template
+:comment: ### Indication of Closing date for voting
+:vote-ended-date: 2020-01-01
+:doctype: cover-page
```
Comment (opoudjis) [inline cover_page/document.adoc]: There is no such doctype; this should be the doctype of international-standard, as the cover page metadata is just affixed to such a document.

**Review by opoudjis — CHANGES_REQUESTED** [review]

File: cover_page/document.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply cover_page/document.adoc]: Thanks @opoudjis, I have corrected it.
Do you think is there anything else to correct or enhance?

**Comment by ronaldtse** [comment]
Re-running jobs for new Fontist that supports Ruby 2.4.

**Review by opoudjis — APPROVED** [review]

**Comment by ronaldtse** [comment]
Thank you @manuel489 !

**Pull request merged**